data/translations/pt_PT.ts

```diff
@@ -2,6 +2,20 @@
 <!DOCTYPE TS>
 <TS version="2.1" language="pt_PT">
 <context>
+    <name>PictureBox</name>
+    <message>
+        <source>Press to login</source>
+        <translation type="unfinished"></translation>
```

Comment:
there is a build failure now:

https://travis-ci.org/sddm/sddm/jobs/255215522#L1208